_study-guides/2017-03-19-sister-outsider-study-guide.md

```diff
@@ -1,10 +1,6 @@
 ---
-layout: posts
-title: "Sister Outsider by Audre Lorde"
-slug: "sister-outsider"
-date:   2017-03-19
-categories: study-guides
-tags: feminism anti-racism anti-capitalism
+identifier: sister-outsider
+slug: sister-outsider
```

Comment:
The slug value should be wrapped in double-quotes. Currently the link is interpolated from the filename.